src/frontends/pytorch/src/op/fake_quantize.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/fake_quantize.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/maximum.hpp"
+#include "openvino/op/minimum.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/scatter_elements_update.hpp"
+#include "openvino/op/select.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_fake_quantize_per_tensor_affine(const NodeContext& context) {
+    num_inputs_check(context, 4, 5);
+    auto input_node = context.get_input(0);
+    auto scale = std::make_shared<v0::Convert>(context.get_input(1), element::f32);
+    auto zero_point = std::make_shared<v0::Convert>(context.get_input(2), element::f32);
+    auto out_low_input = context.get_input(3);
+    auto out_high_input = context.get_input(4);
+    auto out_low = std::make_shared<v0::Convert>(out_low_input, element::f32);
+    auto out_high = std::make_shared<v0::Convert>(out_high_input, element::f32);
+    // Set default levels value to 256. This would be correct for bounds like: 0-255 and -128-127
+    int levels = 256;
+    if (std::dynamic_pointer_cast<v0::Constant>(out_low_input.get_node_shared_ptr()) &&
```

Comment:
I think that should be an exception if we cant calculate levels when inputs are not constants

Reply:
Also please use get_input_from_visible_context function to get constants, or you can even get inputs 4 and 5 as constant values from the very beginning, because if they aren't constant we can not represent them as openvino FakeQuantize anyway